internal/action/sync.go

```diff
@@ -100,15 +100,15 @@ func (s *Action) syncMount(ctx context.Context, mp string) error {
 		debug.Log("Push succeeded")
 		out.Printf(ctxno, color.GreenString("OK"))
 	case errors.Is(err, store.ErrGitNoRemote):
-		out.Printf(ctxno, "Skipped (no remote)")
+		out.Printf(ctx, "Skipped (no remote)")
```

Comment:
Shouldn't this be printed on stderr instead?
Not sure tho, because I guess we might want it to break people's scripts in case they forget to add a remote?

Reply:
`gopass sync` is an interactive command. Skipping because there is no remote is not an error, just a notice.
I have stores without any remotes. Perfectly legit.